hail_search/hail_search_query.py

```diff
@@ -318,6 +331,39 @@ def _valid_genotype_family_entries(cls, entries, gentoype_entry_indices, genotyp
         )
         return hl.or_missing(is_valid, entries)
 
+    @classmethod
+    def _get_genotype_passes_quality_filter(cls, quality_filter):
+        affected_only = quality_filter.get('affected_only')
+        passes_quality = None
+        for filter_k, value in quality_filter.items():
+            field = cls.GENOTYPE_FIELDS.get(filter_k.replace('min_', ''))
+            if field and value:
+                passes_quality = cls._get_genotype_passes_quality_field(field, value, affected_only, passes_quality)
```

Comment:
Would it be more straightforward to use an array of functions as the following?
```
passes_qualities = []
for ...:
    ...
    passes_qualities.append(cls. _get_genotype_passes_quality_field(...))

passes_quelity = None if not passes_qualities else lambda gt: passes_qualities.all(lambda: field_filter: field_filter(gt))
```
And we don't need the cascading filter_func parameter for cls._get_genotype_passes_quality_field function below.